src/components/profile/header.component.tsx

```diff
@@ -78,20 +78,20 @@ export default function Header({ isAnonymous, user, profile, loading, isGuest }:
   };
 
   const handleUnFriendRequest = () => {
-    cancelFriendRequest(friendStatus);
+    if (friendStatus) cancelFriendRequest(friendStatus);
```

Comment:
mungkin lebih readable kalau 
```suggestion
    if (isFriend) cancelFriendRequest(friendStatus);
```

Reply:
but friendStatus is not boolean type, itu isinya:
`
{
"id": "60de3df28b77cd4adaf2fc0a",
"status": "approved",
"createdAt": "2021-07-01T22:13:06.000Z",
"friendId": "0x5cb52a37da79e5698c31da276f326ceed461b90024a1135648606823f4a7ed62",
"requestorId": "0x700978e3b5215dda80d1e3bcdf44a0caa7e5ed1f004f0c9ea7dbf6381b89393e"
},
`